gui-backend/src/main/java/com/rln/gui/backend/implementation/balanceManagement/cache/AccountCache.java

```diff
@@ -7,12 +7,18 @@
 import com.rln.gui.backend.implementation.balanceManagement.data.AccountInfo;
 import com.rln.gui.backend.implementation.balanceManagement.exception.IbanNotFoundException;
 import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
 import java.util.concurrent.ConcurrentHashMap;
 
 public class AccountCache {
 
     private final Map<String, AccountInfo> accounts = new ConcurrentHashMap<>();
 
+    public Set<Entry<String, AccountInfo>> getAccounts() {
+        return accounts.entrySet();
+    }
```

Comment:
It seems you only need this:
```suggestion
    public Set<String> getAccounts() {
        return accounts.keySet();
    }
```